### PR TITLE
Login tweak

### PR DESCRIPTION
**Pull request by rafi-isakh**
This pull request related to issue #41

### Breakdown:
- [x] Login with mobile phone
- [x] Register with mobile phone

### CODE REVIEW

**Comment by andy-shi88** [comment]
@rafi-isakh is this ready?

**Comment by rafi-isakh** [comment]
Yes, if you feel there is no need for changes, then it's ready @andy-shi88

File: app/containers/Main/index.js [hunk 1]
```diff
@@ -86,9 +92,15 @@ class Main extends Component {
           <View style={styles.registerSection}>
             <Text
               style={styles.registerText}
+<<<<<<< HEAD
```
Comment (andy-shi88) [inline app/containers/Main/index.js]: conflicts

File: app/containers/Register/index.js [hunk 2]
```diff
@@ -100,11 +103,115 @@ class Register extends Component {
...
         error_username,
         error_email,
-        error_password
+        error_password,
+        error_phone
       } = errorFields || false;
 
       return (
         <Container style={styles.container}>
+<<<<<<< HEAD
```
Comment (andy-shi88) [inline app/containers/Register/index.js]: conflicts

File: app/containers/Register/reducer.js [hunk 3]
```diff
@@ -18,11 +18,20 @@ import { fromJS } from 'immutable';
 const initialState = fromJS({
   inputFields: {},
   errorFields: {
+<<<<<<< HEAD
```
Comment (andy-shi88) [inline app/containers/Register/reducer.js]: conflicts

**Review by andy-shi88 — CHANGES_REQUESTED** [review]
seems like there are some conflicts @rafi-isakh  can you solve?

**Comment by rafi-isakh** [comment]
resolving conflict done @andy-shi88

**Review by andy-shi88 — APPROVED** [review]

**Comment by andy-shi88** [comment]
thanks @rafi-isakh

**Pull request merged**